cli/cmd/report_definitions.go

```diff
@@ -160,6 +167,64 @@ var (
 	}
 )
 
+func buildReportDefinitionTable(reportDefinition api.ReportDefinition) {
+	headers := [][]string{
+		{reportDefinition.ReportDefinitionGuid, reportDefinition.ReportName, reportDefinition.ReportType,
+			reportDefinition.SubReportType},
+	}
+
+	cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "TYPE", "SUB-TYPE"}, headers))
+	cli.OutputHuman("\n")
+	cli.OutputHuman(buildReportDefinitionDetailsTable(reportDefinition))
+}
+
+func fetchReportDefinitionVersion(id string) error {
+	var (
+		err              error
+		version          int
+		reportDefinition api.ReportDefinition
+	)
+
+	// if no version is supplied return all previous versions
+	if reportDefinitionsCmdState.Version == "all" {
+		cli.StartProgress("Fetching all report definition versions...")
+		response, err := cli.LwApi.V2.ReportDefinitions.GetVersions(id)
+		cli.StopProgress()
+
+		if err != nil {
+			return err
+		}
+
+		return cli.OutputJSON(response)
+	}
+
+	if version, err = strconv.Atoi(reportDefinitionsCmdState.Version); err != nil {
+		return errors.Wrap(err, "unable to parse version")
+	}
+
+	cli.StartProgress(fmt.Sprintf("Fetching report definition version %d...", version))
+	response, err := cli.LwApi.V2.ReportDefinitions.GetVersions(id)
```

Comment:
nit - If I run:
```
lacework rd show <id> --version 3432
```
I would get an error from the APIs saying that it does not exist (I imagine), but it would be better error message to say also which versions actually exist.